lib/views.js

```diff
@@ -384,19 +384,15 @@ exports.tasks_pending = {
      *  message needs the `to` and `message` properties
      */
     function hasPending(tasks) {
-      var tasks = tasks || [];
-
-      tasks.forEach(function(task) {
+      return tasks && tasks.some(function(task) {
         if (task && task.state === 'pending') {
-          task.messages.forEach(function(msg) {
+          return task.messages && task.messages.some(function(msg) {
             if (msg && msg.to && msg.message) {
               return true;
```

Comment:
What happened to the fix requested at https://github.com/medic/medic-webapp/pull/2599/commits/218049e767d9808fd6c3fcd2e7fc2ad565b051f6#diff-f67e868f449edebd4bf847800c170d36L399?